pyterrier/bootstrap.py

```diff
@@ -271,14 +271,55 @@ def _index_add(self, other):
             raise ValueError("Cannot document-wise merge indices with and without positions (%r vs %r)" % (blocks_1, blocks_2))
         multiindex_cls = autoclass("org.terrier.realtime.multi.MultiIndex")
         return multiindex_cls([self, other], blocks_1, fields_1 > 0)
+    
+    def _index_corpusiter(self, direct=True):
```

Comment:
`direct` isn't the most intuitive name to me for this field. Perhaps `return_toks`?

Reply:
or `toks`? The corresponding indexing arg is `pretokenised`?